test/Sema/enum_equatable_hashable.swift

```diff
@@ -12,9 +12,9 @@ var aHash: Int = Foo.A.hashValue
 enum Generic<T> {
   case A, B
 
-  func method() -> Int {
+  static func method() -> Int {
     if A == B { }
-    return A.hashValue
+    return Generic.A.hashValue
```

Comment:
This change was based on the comment by @jrose-apple on a previous PR: https://github.com/apple/swift/pull/2224#discussion_r62131572

But I can change it to be non-static if you think that’s better